src/globus_cli/commands/flows/run/resume.py

```diff
@@ -68,6 +62,41 @@ def resume_command(
     display(res, fields=fields, text_mode=TextMode.text_record)
 
 
+def check_inactive_reason(
+    login_manager: LoginManager,
+    gare: GlobusAuthRequirementsError | None,
+) -> None:
+    if gare is None:
+        return
+    if gare.authorization_parameters.required_scopes:
+        consent_required = not _has_required_consent(
+            login_manager, gare.authorization_parameters.required_scopes
+        )
+        if consent_required:
+            raise CLIAuthRequirementsError(
+                "This timer is missing a necessary consent in order to resume.",
+                gare=gare,
+            )
+
+    # at this point, the required_scopes may have been checked and satisfied
+    # therefore, we should check if there are additional requirements other than
+    # the scopes/consents
+    unhandled_requirements = set(gare.authorization_parameters.to_dict().keys()) - {
```

Comment:
No need for a call to `.keys()` because `set(dict_instance)` returns a set of keys.

```suggestion
    unhandled_requirements = set(gare.authorization_parameters.to_dict()) - {
```